src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/EnumConverter.cs

```diff
@@ -115,6 +116,7 @@ private static long GetEnumValue(bool isUnderlyingTypeUInt64, object enumVal, Cu
         /// <summary>
         /// Converts the given value object to the specified destination type.
         /// </summary>
+        [UnconditionalSuppressMessage("Trimming", "IL2075:", Justification = "Trimmer does not trim enums")]
```

Comment:
These suppressions are on very long methods. We prefer as targeted suppressions as possible. E.g. `[UnconditionalSuppress] FieldInfo GetPublicStaticEnumField(string name) => EnumType.GetField(name, Public | Static)`.

Reply:
Thanks!